bots/zephyr_mirror_backend.py

```diff
@@ -1004,6 +1040,7 @@ def parse_args():
     return parser.parse_args()
 
 def die_gracefully(signal, frame):
+    # type: (Any[dict[[int, type], int]], object) -> None
```

Comment:
signal is probably a Union type rather than an any type.
frame should be a frame object (see stack frame)